.github/workflows/build.yml

```diff
@@ -226,10 +222,26 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Install process user data file script dependencies
+        run: python -m pip install boto3 jinja2
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the Windows admin password
+        # pulled from AWS Parameter Store. Return the admin password to
+        # pass into Packer as an environment variable.
+        run: |
+          output=$(python .github/scripts/process_user_data_file.py)
+          echo "::add-mask::$output"
+          echo "::set-output name=pass::$output"
       - name: Create machine image
         # This runs through the AMI creation process but does not
-        # actually create an AMI
+        # actually create an AMI.
+        # See https://www.packer.io/guides/hcl/variables under the
+        # "From environment variables" section for details about
+        # PKR_VAR environment variables
         run: |
+          PKR_VAR_winrm_password=\
+          "${{ steps.process-user-data.outputs.pass }}" \
           packer build -timestamp-ui \
             -var skip_create_ami=true \
```

Comment:
```suggestion
          packer build -timestamp-ui \
            -var skip_create_ami=true \
            -var winrm_password=${{ steps.process-user-data.outputs.pass }} \
```

Reply:
@mcdonnnj please refer to this comment: https://github.com/cisagov/windows-server-packer/pull/10#discussion_r876577012

Reply:
resolved with [e3b38fe](https://github.com/cisagov/windows-server-packer/pull/10/commits/e3b38fee3199914b48256f154ebff8382bc9701d)